modules/kubecf/install.sh

```diff
@@ -73,11 +86,16 @@ wait_for "kubectl get crd boshdeployments.quarks.cloudfoundry.org -o name"
 info "Test CRDs are ready"
 #wait_for "kubectl apply -f ../kube/cf-operator/boshdeployment.yaml --namespace=scf"
 wait_for "kubectl apply -f ../kube/cf-operator/password.yaml --namespace=scf"
-wait_for "kubectl apply -f ../kube/cf-operator/qstatefulset_tolerations.yaml --namespace=scf"
+if [[ "${DOCKER_REGISTRY}" != "registry.suse.com" ]]; then
```

Comment:
Shouldn't this be as follows? as we don't want to wait if we change the registry, as it will fail.
```suggestion
if [[ "${DOCKER_REGISTRY}" == "registry.suse.com" ]]; then
```